vm/devices/net/netvsp/src/lib.rs

```diff
@@ -2785,12 +2803,13 @@ impl<T: RingMem> NetChannel<T> {
                 tracing::trace!(?request, "handling control message MESSAGE_TYPE_SET_MSG");
 
                 let status = match self.adapter.handle_oid_set(primary, request.oid, reader) {
-                    Ok(restart_endpoint) => {
+                    Ok((restart_endpoint, stop_rx)) => {
                         // Restart the endpoint if the OID changed some critical
                         // endpoint property.
                         if restart_endpoint {
                             self.restart = Some(CoordinatorMessage::Restart);
                         }
+                        self.stop_rx = stop_rx;
```

Comment:
Don't we also need to do something to restart processing of the subchannels with the new value?